server/settings/create.py

```diff
@@ -9,6 +9,15 @@ class CreatorModel(BaseSettingsModel):
         default_factory=list,
     )
 
+
+class CreatorModelFBX(BaseSettingsModel):
+    enabled: bool = SettingsField(title="Enabled")
+    default_variants: list[str] = SettingsField(
+        title="Default Products",
+        default_factory=list,
+    )
+
+
```

Comment:
In this case this extra settings model is not needed. It's exactly like the one above -and there isn't a reason for a new unique on here. Note also how the other settings entries below, multiples of them use the `CreatorModel`. It's like the base model for Creators, it's not specific to the model creator. ;)

```suggestion```